utils/validateLspMetadata.ts

```diff
@@ -69,9 +79,25 @@ export const validateLsp4Metadata = (
  * @param image - image object to be validated
  * @returns - true if validation passes, false otherwise
  */
-const validateImage = (image: any) => {
-  // we are ok if there is at least url
-  return image?.url
+export const validateImage = (images: unknown): ImageMetadata[] => {
+  return (
+    (Array.isArray(images) &&
+      images?.filter(image => {
+        const imageCheck =
+          'url' in image ||
+          ('url' in image &&
+            'verification' in image &&
+            'verification.data' in image &&
+            'verification.method' in image)
+
+        if (!imageCheck) {
+          console.warn('Invalid LSP4 image metadata', image)
```

Comment:
But it's important to know whether it's an LSP3 or LSP4 image rather than the fact it ultimately was stored as a VerifiableUri, hmmm.

Reply:
Then `validateImage` should get an extra argument specifying the standard because right now it's used for both LSP3 (lines 25,28) and LSP4 (line 60) but prints only LSP4.

Reply:
Tbh we don't need this warning so much and it just make a lot of console noise. I'll simplify this.